static/app/types/dynamicSampling.tsx

```diff
@@ -49,13 +49,15 @@ export enum DynamicSamplingInnerName {
   TRACE_ENVIRONMENT = 'trace.environment',
   TRACE_USER_ID = 'trace.user.id',
   TRACE_USER_SEGMENT = 'trace.user.segment',
+  TRACE_TRANSACTION = 'trace.transaction',
   EVENT_RELEASE = 'event.release',
   EVENT_ENVIRONMENT = 'event.environment',
   EVENT_USER_ID = 'event.user.id',
   EVENT_USER_SEGMENT = 'event.user.segment',
   EVENT_LOCALHOST = 'event.is_local_ip',
   EVENT_WEB_CRAWLERS = 'event.web_crawlers',
   EVENT_BROWSER_EXTENSIONS = 'event.has_bad_browser_extensions',
+  EVENT_TRACE = 'event.trace',
```

Comment:
```suggestion
  EVENT_TRACE = 'event.transaction',
```

@priscilawebdev I think this name is different according to:
https://github.com/getsentry/relay/blob/68bb37683319a3611755810853b715373d09163d/relay-sampling/src/lib.rs#L368

and https://github.com/getsentry/relay/pull/1058

Reply:
@priscilawebdev `EVENT_TRACE` should be named to `EVENT_TRANSACTION` since this is referring to the event's transaction name. 

The event's trace has a different meaning. It refers to the trace id.

Reply:
oh I see...I believe there is a typo in the documentation that Radu created to describe the API 

![image](https://user-images.githubusercontent.com/29228205/130744518-ae9369ee-1d00-42f8-baf0-dff5a64d3a46.png)


I'll fix it here in the code too... Thanks @dashed 🙏